modules/API/nav.adoc

```diff
@@ -1,3 +1,4 @@
 * xref:index.adoc[REST API Reference]
 ** xref:gsql-endpoints.adoc[GSQL Endpoints]
-** xref:upsert-rest.adoc[Upsert Data (3.x)]
\ No newline at end of file
+** xref:upsert-rest.adoc[Upsert Data (3.x)]
```

Comment:
we may not need to mark it as (3.x), since 4.1 still valid

Reply:
Done